### DISCUSSION TITLE
About `qiskit_trebugger`

### DISCUSSION THREAD

**Discussion by RIvance** [Qiskit/.github#29] [General Feedback]
The qiskit transpiler debugger really is an amazing feature, especially for those who are interested in circuit optimization. But the `Transpiler Passes` widget is too small to show a full image of a long circuit. It is really inconvenient when debugging complex quantum algorithms. It would be better if you increase the height of the widget or just make the widget scalable.
![Screenshot from 2023-03-16 18-40-56](https://user-images.githubusercontent.com/72872760/225594590-c8173b87-0dc5-4b9b-9517-977e77d016ed.png)

**Comment by kdk** [comment]
Thanks for the feedback @RIvance ! @TheGupta2012 is continuing work on the transpiler debugger, pinging for his awareness.

**Reply by TheGupta2012** [reply]
Thanks @RIvance for the feedback! 

There is an option to download this image which may help in better viewing.

Moreover, we are working on improving the widget this spring and would try to incorporate your feedback too.

**Reply by RIvance** [reply]
Thanks for the reply! I'm looking forward to the trebugger getting better and better.

**Comment by 1ucian0** [comment]
For reference, [here is the qiskit-timeline-debugger](https://github.com/TheGupta2012/qiskit-timeline-debugger).
